src/components/Calculator.js

```diff
@@ -1,40 +1,56 @@
 import React from 'react';
 import './Calculator.css';
+import calculate from '../logic/calculate';
 
 export default class Calculator extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
+
+  handleEvent = (event) => {
+    const result = calculate(this.state, event.target.textContent);
+    this.setState(result);
+  }
+
   render() {
+    const { total, next } = this.state;
     return (
       <>
         <div className="calculator">
-          <input className="calculator-input" type="input" placeholder="0" />
+          <input className="calculator-input" type="input" placeholder={next || total} />
```

Comment:
- [OPTIONAL] I can see that you are using the `<input>` tag here to show the results that's not bad 👌 but using the input tag allows the user to write any value inside that input and we don't want that, so I suggest using the `readOnly` attribute to prevent the user from writing any value and making The input is read-only 👌💯

> For Example

```
          <input
            className="calculator-input"
            type="input"
            placeholder={next || total}
            readOnly
          />
```

> the issue
![image](https://user-images.githubusercontent.com/45231276/184269559-ca29fe21-459f-4c23-8293-a372b6ba0607.png)